files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md

```diff
@@ -1,77 +1,79 @@
 ---
 title: Error.prototype.toString()
 slug: Web/JavaScript/Reference/Global_Objects/Error/toString
+l10n:
+  sourceCommit: 27180875516cc311342e74b596bfb589b7211e0c
 ---
 
 {{JSRef}}
 
-**`toString()`** 方法返回一个表示指定 {{jsxref("Error")}} 对象的字符串。
+{{jsxref("Error")}} 实例的 **`toString()`** 方法返回一个表示此错误的字符串。
 
 ## 语法
 
 ```js-nolint
 toString()
 ```
 
+### 参数
+
+无。
+
 ### 返回值
 
 一个表示指定 {{jsxref("Error")}} 对象的字符串。
 
 ## 描述
 
-{{jsxref("Error")}} 对象覆盖了 {{jsxref("Object.prototype.toString()")}} 方法。该方法实现如下（假定 {{jsxref("Object")}} 和 {{jsxref("String")}} 没有被更改）：
+{{jsxref("Error")}} 对象重写了所有对象继承的 {{jsxref("Object.prototype.toString()")}} 方法。其语义如下：
 
 ```js
 Error.prototype.toString = function () {
-  "use strict";
-
-  const obj = Object(this);
-  if (obj !== this) {
+  if (
+    this === null ||
+    (typeof this !== "object" && typeof this !== "function")
+  ) {
     throw new TypeError();
   }
-
   let name = this.name;
-  name = name === undefined ? "Error" : String(name);
-
+  name = name === undefined ? "Error" : `${name}`;
   let msg = this.message;
-  msg = msg === undefined ? "" : String(msg);
-
+  msg = msg === undefined ? "" : `${msg}`;
   if (name === "") {
     return msg;
   }
   if (msg === "") {
     return name;
   }
-
-  return name + ": " + msg;
+  return `${name}: ${msg}`;
 };
 ```
 
 ## 示例
 
-### 使用 toString()
+### 使用 toString() 方法
```

Comment:
Please be consistent with the English documentation. There is no disambiguation here and it's not necesssary to add the "method".

```suggestion
### 使用 toString()
```